ai_diffusion/workflow.py

```diff
@@ -223,6 +267,12 @@ def load_model_with_lora(w: ComfyWorkflow, comfy: Client, style: Style, is_live=
             continue
         model, clip = w.load_lora(model, clip, lora["name"], lora["strength"], lora["strength"])
 
+    for lora in additional_loras:
```

Comment:
This could be merged with the loop above, right?
```
for lora in chain(style.loras, additional_loras):
    ...
```

Reply:
The only difference was the log output, it's not that important. I'll merge the loops.